### ISSUE TITLE
How to authenticate from server side?

### ISSUE THREAD

**Issue by lukedukeus** [Azure-Samples/ms-identity-javascript-react-tutorial#143]
I have a react app with a ASP.NET server. The server just sends the static files to the client, but there are two versions of the client. I want to switch which version the client gets based off of whether the user is authenticated or not. 

One way I could implement this is by doing authentication on the client side, then passing the authentication token to the server, which determines which static files to return. 

Another way I could implement this is by doing authentication server side, and add a controller with routes to sign in / out. This way would be easier for me to implement, because then when the user calls any other route, the authentication is injected.

However, I can't find any examples on how to implement either way! All the samples from [here](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2) show server side authentication, but the client is written in cshtml, and the webapi examples are missing routes to login / out.

All the samples [here](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial) are missing the ASP.NET Backend. 
Does anyone know the authentication flow I am looking for, or where to find an example of it?

**Comment by salman90** [comment]
Hi @lukedukeus,

Sorry for the late response. From my understanding, you're trying to find a sample that uses ASP.NET on the backend and JavaScript on the frontend. Please check the following sample [2-5-HybridFlow](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/tree/master/2-WebApp-graph-user/2-5-HybridFlow).

**Comment by lukedukeus** [comment]
No prob, I'm just glad to get any response! 
From what I have learned, client and server can't share users, they can only pass auth tokens between each other. Because of this, I moved all the authentication to the server side, and it works, but you miss out on account switching. See this [question](https://stackoverflow.com/questions/73614801/how-to-support-multiple-signed-in-users-in-asp-net-core).
